test/pom.xml

```diff
@@ -135,7 +135,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>structs</artifactId>
-      <version>1.23</version>
+      <version>1.24-rc301.b358e327c0ec</version> <!-- TODO https://github.com/jenkinsci/structs-plugin/pull/112 -->
```

Comment:
This must be in draft so long as that is unreleased, since the incremental build could be collected at some point and we would not want this dep in `master`. See https://github.com/jenkinsci/pom/pull/172.

Reply:
this could be separated out to another PR I guess.